Jellyfin.Server/CoreAppHost.cs

```diff
@@ -78,7 +78,8 @@ protected override void RegisterServices()
             }
 
             ServiceCollection.AddDbContextPool<JellyfinDb>(
-                 options => options.UseSqlite($"Filename={Path.Combine(ApplicationPaths.DataPath, "jellyfin.db")}"));
+                 options => options.UseLoggerFactory(LoggerFactory).
+                    UseSqlite($"Filename={Path.Combine(ApplicationPaths.DataPath, "jellyfin.db")}"));
```

Comment:
```suggestion
                 options => options
                   .UseLoggerFactory(LoggerFactory)
                   .UseSqlite($"Filename={Path.Combine(ApplicationPaths.DataPath, "jellyfin.db")}"));
```

I'm sure I got indentation incorrect, but this is more readable

Reply:
Good catch!